lib/oeqa/utils/helper.py

```diff
@@ -64,7 +64,7 @@ def get_native_dir():
 def add_group(groupname, gid=None, target=None):
     target = target if target is not None else oeRuntimeTest.tc.target
     (ret, output) = target.run("groupadd %s %s"%(groupname, "" if gid is None else "-g %s"%gid))
-    print >> sys.stderr, output
+    print (output, file=sys.stderr)
```

Comment:
This will most likely not work with Python2, where print() is just the regular print statement applied to a tuple.

You can fix this with (at the top of the file, before other import statements):
```
from __future__ import print_function
```
```